tests/test_lsp/test_lsp.py

```diff
@@ -0,0 +1,227 @@
+"""Test Sphinx-Needs language features."""
+
+import os
+import sys
+
+import pytest
+import pytest_lsp
+from pygls.lsp.types import MarkupContent, MarkupKind, Position, Range
+from pytest_lsp import ClientServerConfig
```

Comment:
`pytest_lsp` got already imported.

So use `pytest_lsp.ClientServerConfig` in your code